doc/adr/0012-remove-snyk.md

```diff
@@ -0,0 +1,24 @@
+# 12. remove snyk
+
+Date: 2018-03-26
+
+## Status
+
+Accepted
+
+## Context
+
+Many of the vulnerabilities identified by snyk have no fixes, and can only be ignored in the synk config file for a set number of days.
+After the number of days are up, the vulnerabilities will fail the build and prevent merge on new PRs, with the only option to ignore the
+vulnerabilities again.
+
+## Decision
```

Comment:
I'd say the first 2 sentences are providing more context.
The last sentence needs a bit of work, perhaps adding `Removing Snyk in place of a combination...` would do it.